data_prep/introspector.py

```diff
@@ -49,15 +50,16 @@
 
 def set_introspector_endpoints(endpoint):
   """Sets URLs for Fuzz Introspector endpoints to local or remote endpoints."""
-  global INTROSPECTOR_ENDPOINT, INTROSPECTOR_CFG, INTROSPECTOR_FUNCTION, \
-      INTROSPECTOR_SOURCE, INTROSPECTOR_XREF, INTROSPECTOR_TYPE, \
-      INTROSPECTOR_FUNC_SIG
+  global INTROSPECTOR_ENDPOINT, INTROSPECTOR_CFG, INTROSPECTOR_FUNC_SIG, \
+      INTROSPECTOR_SOURCE, INTROSPECTOR_XREF, INTROSPECTOR_TYPE,         \
```

Comment:
Thanks, @DavidKorczynski, this could be an auto-formatting issue:
Are the spaces between the last `,` and `\` added by auto-formatter?
Just want to double-check that our auto-formatting script does not get in your way : )

Reply:
> Are the spaces between the last , and \ added by auto-formatter?

No, I added them to keep the `\` aligned.

Running prehooks give me:

```sh
(.venv) $ .github/helper/presubmit
PYLINTHOME is now '/home/dav/.cache/pylint' but obsolescent '/home/dav/.pylint.d' is found; you can safely remove the latter
************* Module introspector
data_prep/introspector.py:61:0: C0301: Line too long (87/80) (line-too-long)
data_prep/introspector.py:62:0: C0301: Line too long (89/80) (line-too-long)
data_prep/introspector.py:162:4: W1201: Use lazy % formatting in logging functions (logging-not-lazy)

added 1 package, and audited 2 packages in 2s

found 0 vulnerabilities
0 errors, 0 warnings, 0 informations 
```

Reply:
> No, I added them to keep the \ aligned.

Ah I see, we don't have to keep it aligned. Could you please remove it, if you don't mind? Thanks!

> data_prep/introspector.py:61:0: C0301: Line too long (87/80) (line-too-long)
data_prep/introspector.py:62:0: C0301: Line too long (89/80) (line-too-long)

Would you mind fixing these two, a potential solution is using `()`, e.g.:
```python
INTROSPECTOR_ORACLE_FAR_REACH = (f'{INTROSPECTOR_ENDPOINT}/'
                                 'far-reach-but-low-coverage')
``` 
> logging.error('No such oracle "%s"' % (target_oracle))

I think the formatter was suggesting:
```python
logging.error('No such oracle "%s"', target_oracle)
```

Reply:
Yep, done! Can confirm:

```sh
$ .github/helper/presubmit
0 errors, 0 warnings, 0 informations
```

Reply:
Amazing, thanks!